a10_octavia/controller/worker/tasks/glm_tasks.py

```diff
@@ -124,16 +136,31 @@ def revert(self, vthunder, flavor=None, *args, **kwargs):
 
 class ActivateFlexpoolLicense(task.Task):
 
+    def build_configuration(self, flavor={}):
```

Comment:
we could use static method instead of instance method.

Reply:
We could but what's the benefit over a class method? A static method is generally used as a utility when you call the class. This is being used internally. If anything, we should turn this into a protected method since it's internal to the class.

Reply:
This method does not use the object instance i.e, self or class  for its operation. I think there is no use accepting self parameter for this utility function.  If turned to static we can invoke it through class and object as well.

Reply:
Right by why would you do that? We don't ever use it via the class